jsp/build.xml

```diff
@@ -52,29 +56,58 @@
         <mkdir dir="${build.dir}"/>
         <mkdir dir="${classes.dir}"/>
         <mkdir dir="${test.classes.dir}"/>
+        <mkdir dir="${test.reports.dir}"/>
         <mkdir dir="${ivy.reports.dir}"/>
     </target>
 
     <target name="compile" depends="init">
+        <ivy:retrieve pattern="${dist.lib}/[artifact]-[revision].[ext]" conf="runtime"/>
         <javac srcdir="${src.dir}" destdir="${classes.dir}" source="1.6" target="1.6" includeantruntime="no" encoding="UTF-8" debug="true" classpathref="runtime.path">
             <include name="**/*.java"/>
+            <exclude name="${test.src.path}/*.java"/>
         </javac>
     </target>
 
     <target name="clean">
         <delete failonerror="false" includeEmptyDirs="true">
             <fileset dir="${external.dir}"/>
             <fileset dir="${ivy.reports.dir}"/>
+            <fileset dir="${test.reports.dir}"/>
             <fileset dir="${test.classes.dir}"/>
             <fileset dir="${classes.dir}"/>
             <fileset dir="${build.dir}"/>
             <fileset file="${dist.jar}"/>
         </delete>
     </target>
 
+    <target name="test-junit" depends="init">
+        <ivy:retrieve pattern="${dist.lib}/[artifact]-[revision].[ext]" conf="test"/>
+        <javac srcdir="${src.dir}" destdir="${test.classes.dir}" source="1.6" target="1.6" includeantruntime="no" encoding="UTF-8" classpathref="test.path">
```

Comment:
Should we update the source and target to 1.8 in this and line 65?

Reply:
Ok, updating the version was omitted. thank you.